src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_attend_and_excite.py

```diff
@@ -762,6 +763,8 @@ def __call__(
                 Dictionary defining the iterations and desired thresholds to apply iterative latent refinement in.
             scale_factor (`int`, *optional*, default to 20):
                 Scale factor that controls the step size of each Attend and Excite update.
+            attn_res (`int`, *optional*, default to 16):
+                The resolution of most semantic attention map.
```

Comment:
Is this something that could be changed during inference or not?

Reply:
technically yes but I don't think it's a common use case to change 

Basically, you would want to apply attend-and-excite on the most semantic attention resolutions. The paper was based on SD 1.5, and it used 16x16.  But if users want to use it on a different model, they might want to test it on different resolutions to find it out.